packages/design-tokens/scripts/formats.js

```diff
@@ -23,10 +23,10 @@ const colorsForStyleguide = {
 };
 
 const commonJS = {
-  name: "javascript/commonJS",
+  name: "typescript/es6",
   formatter: dictionary =>
     `${dateHeader()}\n${dictionary.allProperties
-      .map(prop => `exports.${prop.name} = '${prop.value}';`)
+      .map(prop => `export const ${prop.name}: string = "${prop.value}";`)
```

Comment:
Setting specific types for each category of design tokens felt like overkill, so I'm just using the `string` type for everything.

I can be more specific if anybody disagrees